core/include/traccc/clusterization/clusterization_algorithm.hpp

```diff
@@ -8,6 +8,7 @@
 #pragma once
 
 // Library include(s).
+#include "traccc/clusterization/clustering_config.hpp"
```

Comment:
Optional: Maybe worth removing this include, as it's not needed in this specific header.

Reply:
:+1: